platform-sdk/docs/core/platform-status.md

```diff
@@ -0,0 +1,65 @@
+# Platform Status
+
+The platform status is represented as an enum value from `PlatformStatus.java`. It is updated via
+`SwirldsPlatform::checkPlatformStatus`, which is passed around as a callback to any object that has the ability to
+effect a change in platform status.
+
+A diagram of expected state transitions is available [here](./platform-status-transitions.svg). Note that there are
+currently no hard restrictions on state transitions: this diagram is merely a representation of the expected behavior.
+
+## Used Statuses
+
+The following statuses, along with their respective triggers, are listed in order of precedence (from highest to lowest):
+
+- `DISCONNECTED`: There is more than one node in the network, and the number of active connections is zero
+- `BEHIND`: The sync manager has reported that the node has fallen behind
+- `MAINTENANCE`: The freeze manager has reported that a freeze has started
```

Comment:
This README is to capture the current behavior - a better solution will be created for https://github.com/hashgraph/hedera-services/issues/6296. I'm not sure if a maintenance mode would make sense in general, but I agree it's poorly named currently

Reply:
At some point we should rename `MAINTENANCE` to `FREEZE_STARTED`. We didn't do it before because a lot of JRS tests depend on this marker. But we should still do it at some point.